apps/api/src/app/notification-template/notification-template.controller.ts

```diff
@@ -148,6 +153,50 @@ export class NotificationTemplateController {
     );
   }
 
+  @Get('/:templateId/blueprint')
+  @ApiOkResponse({
+    type: NotificationTemplateResponse,
+  })
+  @ApiOperation({
+    summary: 'Get blueprint notification template',
+  })
+  @ExternalApiAccessible()
+  getNotificationTemplateBlueprintById(
+    @UserSession() user: IJwtPayload,
+    @Param('templateId') templateId: string
+  ): Promise<NotificationTemplateResponse> {
+    return this.getBlueprintNotificationTemplate.execute(
+      GetBlueprintNotificationTemplateCommand.create({
+        environmentId: user.environmentId,
+        organizationId: user.organizationId,
+        userId: user._id,
+        templateId,
+      })
+    );
+  }
+
+  @Post('/:templateId/blueprint')
+  @ApiOkResponse({
+    type: NotificationTemplateResponse,
+  })
+  @ApiOperation({
+    summary: 'create notification template from blueprint',
+  })
+  @ExternalApiAccessible()
```

Comment:
I think we can remove those endpoints from public API. What do you think?

Reply:
the remove one yes and I guess the get as well because Nevo can use the normal get template endpoint